src/utils/IdealGraphVisualizer/View/src/main/java/com/sun/hotspot/igv/view/DiagramScene.java

```diff
@@ -1312,34 +1223,24 @@ public void undo() throws CannotUndoException {
             scene.getModel().setData(oldModel);
             scene.getModel().getViewChangedEvent().removeListener(this);
 
-            SwingUtilities.invokeLater(new Runnable() {
-
-                @Override
-                public void run() {
-                    scene.setScrollPosition(oldScrollPosition);
-                }
-            });
+            SwingUtilities.invokeLater(() -> scene.setScrollPosition(oldScrollPosition));
 
             scene.setUndoRedoEnabled(b);
         }
 
         @Override
         public void changed(DiagramViewModel source) {
             scene.getModel().getViewChangedEvent().removeListener(this);
-            if (oldModel.getHiddenNodes().equals(newModel.getHiddenNodes())) {
-                scene.smallUpdate(false);
-            } else {
-                scene.smallUpdate(true);
-            }
+            scene.smallUpdate(!oldModel.getHiddenNodes().equals(newModel.getHiddenNodes()));
         }
     }
     private boolean undoRedoEnabled = true;
 
-    public void setUndoRedoEnabled(boolean b) {
-        this.undoRedoEnabled = b;
+    private void setUndoRedoEnabled(boolean enable) {
+        undoRedoEnabled = enable;
     }
 
-    public boolean getUndoRedoEnabled() {
+    private boolean getUndoRedoEnabled() {
         return undoRedoEnabled;
     }
```

Comment:
You could directly modify the field instead of going over the private setter/getter.

Reply:
done